src/lints/function_parameter_count_changed.ron

```diff
@@ -51,4 +53,7 @@ SemverQuery(
     },
     error_message: "A publicly-visible function now takes a different number of parameters.",
     per_result_error_template: Some("{{join \"::\" path}} now takes {{current_parameter_count}} parameters instead of {{old_parameter_count}}, in {{span_filename}}:{{span_begin_line}}"),
+    witness: (
+    hint_template: r#"let witness = |{{join ", " old_parameter_names}}| {{join "::" path}}({{join ", " old_parameter_names}});"#,
```

Comment:
We don't have an autoformatter for RON files, so please try to be mindful of indentation and formatting by hand. Each nested field should be one more step indented compared to its parent field.
```suggestion
        hint_template: r#"let witness = |{{join ", " old_parameter_names}}| {{join "::" path}}({{join ", " old_parameter_names}});"#,
```